packages/widget-chat/src/containers/notifications/index.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import {Provider} from 'react-redux';
+
+import store from '../../store';
+
+import Notifications from '.';
+
+describe(`Notifications container`, () => {
+  it(`renders properly`, () => {
```

Comment:
I think we wouldn't test the render, but we should probably test the store changes by simulating a notification and making sure everything else fires correctly. Eventually we would want to test if the notification actually gets displayed but I don't think that's possible without an actual browser.